antora-ui-camel/src/css/doc.css

```diff
@@ -368,7 +368,7 @@
 .doc ol,
 .doc ul {
   margin: 0;
-  padding: 0 0 0 2rem;
+  padding: 0 0 0 1rem;
```

Comment:
This change doesn't seem to be related to pagination. If it is the rule needs to be scoped to the pagination and placed in blog.css. Otherwise it affects other `ol` and `ul` elements.